src/component/home/home.component.jsx

```diff
@@ -1,112 +1,115 @@
-import React, { useState, useEffect } from 'react'
-import { useDispatch, useSelector } from 'react-redux'
-import { db } from '../../config/firebase'
-import { logheazaMaAction, scoateMaAction } from '../../store/user.action'
-import { ActionCard } from '../action-card'
-import * as HOS from './home.style'
+import React, { useState, useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { db } from "../../config/firebase";
+import { logheazaMaAction, scoateMaAction } from "../../store/user.action";
+import { AuctionCard } from "../auction-card";
+import * as HOS from "./home.style";
 
 export const Home = () => {
-
-  const [name, setName] = useState('')
-  const [nameList, setNameList] = useState([])
+  const [name, setName] = useState("");
+  const [nameList, setNameList] = useState([]);
 
   const handleChange = (e) => {
-    const {value} = e.target
-    setName(value)
-    console.log(value)
-  }
-//Snapchot for firebase with for each
-  useEffect(
-    () => {
-      db.collection("hahahe").onSnapshot(snap => {
-        let list = []
+    const { value } = e.target;
+    setName(value);
+    console.log(value);
+  };
+  //Snapchot for firebase with for each
+  useEffect(() => {
+    db.collection("hahahe").onSnapshot((snap) => {
+      let list = [];
 
-        snap.forEach((doc) => {
-          list.push({
-            id: doc.id,
-            ...doc.data()
-          })
-        })
-        setNameList(list)
-      })
-    },
-    []
-  )
+      snap.forEach((doc) => {
+        list.push({
+          id: doc.id,
+          ...doc.data(),
+        });
+      });
+      setNameList(list);
+    });
+  }, []);
 
-  const handlePress = e => {
+  const handlePress = (e) => {
     if (e.charCode === 13) {
       // Vreau sa il trimit la baza de date cand apas enter
       db.collection("hahahe").add({
         name,
-        anaaremere: Math.random() - 0.5 > 0
-      })
+        anaaremere: Math.random() - 0.5 > 0,
```

Comment:
Let's leave poor Ana and her apples and lets change this to something that makes a bit more sense in this context :D